### ISSUE TITLE
[Bug] Monitor bar dont show on second monitor when second monitor is on top 

### ISSUE THREAD

**Issue by RodrigoValente97** [glzr-io/zebar#116]
### Describe the bug

Bug present with installer on glazewm 3.2.0  and not with 3.1.1. On windows 11 x86. 
Tried with clean install on zebar V2 and problem is consistent.

### Reproduction

Install on windows 11 and with monitor displayed on top of each other.

### Version number

zebar V2 + galzewn 3.2

**Comment by lars-berger** [comment]
Haven't been able to reproduce this by rearranging my displays, e.g.

![image](https://github.com/user-attachments/assets/f7f0766c-4728-4caf-ba1d-5adb147a6502)

Could you share your display settings? Does it work normally when they're rearranged to be horizontal?

**Comment by Kennethgab** [comment]
I get the same issue with 2x2 setup, my main monitor is 4 here. Zebar wont appear on monitor 1 and 2.
![image](https://github.com/user-attachments/assets/b48d4d26-0d4b-4f9c-87e4-a8c0f158b5ce)

**Comment by lars-berger** [comment]
> I get the same issue with 2x2 setup, my main monitor is 4 here. Zebar wont appear on monitor 1 and 2. ![image](https://private-user-images.githubusercontent.com/21317897/372381814-b48d4d26-0d4b-4f9c-87e4-a8c0f158b5ce.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc3NzI3NTAsIm5iZiI6MTcyNzc3MjQ1MCwicGF0aCI6Ii8yMTMxNzg5Ny8zNzIzODE4MTQtYjQ4ZDRkMjYtMGQ0Yi00ZjljLTg3ZTQtYThjMGYxNThiNWNlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDAxVDA4NDczMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYxMDdmZmExNmEyZGUyYzZlYjYxMTlmM2E2YjRiZTkyYzdlYzlkOWU3NTZmMTc3ZTUzNzI4ODNiMjFhNzIyZDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.t2r-nVTeyxQKA9s6xkrjooCoxhg4b-reUpQr6VVOQNM)

Could you try the [v2.2.2 release](https://github.com/glzr-io/zebar/releases/tag/v2.2.2) and post what output you get from running `zebar query monitors`?

**Comment by Kennethgab** [comment]
> > I get the same issue with 2x2 setup, my main monitor is 4 here. Zebar wont appear on monitor 1 and 2. ![image](https://private-user-images.githubusercontent.com/21317897/372381814-b48d4d26-0d4b-4f9c-87e4-a8c0f158b5ce.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc3NzI3NTAsIm5iZiI6MTcyNzc3MjQ1MCwicGF0aCI6Ii8yMTMxNzg5Ny8zNzIzODE4MTQtYjQ4ZDRkMjYtMGQ0Yi00ZjljLTg3ZTQtYThjMGYxNThiNWNlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDAxVDA4NDczMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYxMDdmZmExNmEyZGUyYzZlYjYxMTlmM2E2YjRiZTkyYzdlYzlkOWU3NTZmMTc3ZTUzNzI4ODNiMjFhNzIyZDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.t2r-nVTeyxQKA9s6xkrjooCoxhg4b-reUpQr6VVOQNM)
> 
> Could you try the [v2.2.2 release](https://github.com/glzr-io/zebar/releases/tag/v2.2.2) and post what output you get from running `zebar query monitors`?

[{"name":"\\\\.\\DISPLAY12","is_primary":false,"x":-3840,"y":0,"width":3840,"height":2160,"scale_factor":1.5},{"name":"\\\\.\\DISPLAY11","is_primary":false,"x":-2560,"y":-1440,"width":2560,"height":1440,"scale_factor":1.0},{"name":"\\\\.\\DISPLAY5","is_primary":false,"x":0,"y":-1440,"width":2560,"height":1440,"scale_factor":1.0},{"name":"\\\\.\\DISPLAY13","is_primary":true,"x":0,"y":0,"width":3840,"height":2160,"scale_factor":1.5}]

**Comment by gpbenton** [comment]
I am having the same problem

[{"name":"\\\\.\\DISPLAY2","is_primary":false,"x":-752,"y":-1440,"width":3440,"height":1440,"scale_factor":1.0},{"name":"\\\\.\\DISPLAY1","is_primary":true,"x":0,"y":0,"width":1920,"height":1080,"scale_factor":1.25}]

although even when the large monitor is on the bottom, the bar appears about 3/4 up the screen, rather than the top, and doesn't have any icons on either edge, just the date in the middle.  I think the icons are probably being put outside the width of the monitor.

**Comment by JonasWischeropp** [comment]
I reproduced it with:
Screen 1 above screen 2.
1 has a smaller resolution than 2.
1 with 100% and 2 with 150%.

**Comment by pu-007** [comment]
![image](https://github.com/user-attachments/assets/04aa26cb-5c19-4b82-9a35-fcf14cafee85)
I also encounter this issue.

**Comment by matsavage** [comment]
I am also having the same problem, it shows when both displays are at the same scale factor, but not when one is 125 and one is 150%

```
zebar query monitors
> [{"name":"\\\\.\\DISPLAY2","is_primary":false,"x":0,"y":-2160,"width":1920,"height":2160,"scale_factor":1.25},
   {"name":"\\\\.\\DISPLAY1","is_primary":true,"x":0,"y":0,"width":3840,"height":2160,"scale_factor":1.5}]
```

![image](https://github.com/user-attachments/assets/86789b4a-dddb-4f1c-9f59-a3eefe15c74d)

**Comment by JonasWischeropp** [comment]
This issue should be fixed by #119.
(Merged, but currently not yet part of any release.)
